scripts/release.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+if [ $# -ne 1 ]; then
+    echo "Usage: $0 <new_version>"
+    exit 1
+fi
+
+NEW_VERSION="$1"
+
+if ! echo "$NEW_VERSION" | grep -E '^[0-9]+\.[0-9]+\.[0-9]+$' >/dev/null; then
+    echo "Error: Version must be in format X.Y.Z (e.g., 1.0.0)"
+    exit 1
+fi
+
+if git rev-parse "v$NEW_VERSION" >/dev/null 2>&1; then
+    echo "Error: Tag v${NEW_VERSION} already exists"
+    exit 1
+fi
+
+perl -i -pe 's/var Version = "[0-9]+\.[0-9]+\.[0-9]+"/var Version = "'"$NEW_VERSION"'"/' evaluate/version.go
```

Comment:
Will rewrite this script to Go. @bauersimon @ruiAzevedo19 please prefer writing such logic in Go. We usually continuously add to processes and Bash is a nightmare to vet, test and maintain.